<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="controlWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>58</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>58</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTimeEdit" name="timeEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string notr="true">mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopAll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playAll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="actionLayerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>96</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="behavioredit.qrc">:/res/image/ic_time_sport.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Action Layer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopAction">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playAction">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="musicLayerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>96</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap resource="behavioredit.qrc">:/res/image/ic_time_music.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Music Layer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopMusic">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playMusic">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="frameControlWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>66</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>66</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="durationLbl">
            <property name="text">
             <string>Action Duration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="totalTime">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="displayFormat">
             <string notr="true">mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="frameRateSettingBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>78</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>78</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Frame</string>
          </property>
          <property name="icon">
           <iconset resource="behavioredit.qrc">
            <normaloff>:/res/image/ic_time_set.png</normaloff>:/res/image/ic_time_set.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="scaleWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="scaleDownBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="behavioredit.qrc">
             <normaloff>:/res/image/ic_time_silde_subtract.png</normaloff>:/res/image/ic_time_silde_subtract.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="scaleSlider">
           <property name="minimumSize">
            <size>
             <width>116</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>116</width>
             <height>14</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="scaleUpBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="behavioredit.qrc">
             <normaloff>:/res/image/ic_time_silde_plus.png</normaloff>:/res/image/ic_time_silde_plus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="behavioredit.qrc"/>
 </resources>
 <connections/>
</ui>
